src/legacy/core_plugins/data/public/query/query_bar/components/typeahead/suggestion_component.tsx

```diff
@@ -19,7 +19,7 @@
 
 import { EuiIcon } from '@elastic/eui';
 import classNames from 'classnames';
-import React, { SFC } from 'react';
+import React, { FunctionComponent } from 'react';
```

Comment:
Just FYI, in case you weren't aware of it so far (if so ignore me): `FunctionComponent` has also a version (like `SFC` beforehand) just called `FC` for people who don't like typing ;-)